docs/UserGuide.md

```diff
@@ -91,8 +92,8 @@ A person can have zero or more roles.
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 r/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com r/athlete`
+* `add n/Betsy Crowe p/98213132 e/betsycrowe@example.com r/volunteer`
```

Comment:
Line 95-96: Can we have properly formatted examples for each special type of role we have?

Reply:
I'll see what I can do